ChangeLog

```diff
@@ -2876,6 +2876,10 @@ Release date: 2003-09-12
 
     * easy functional test infrastructure
 
+	* Raise bad-whitespace error on arithmetic operators that miss spaces
```

Comment:
I have added it in the right location. Is that it? Or should it be added any where else too?